src/jobsautoreport/utils.py

```diff
@@ -1,12 +1,19 @@
 from typing import Any
```

Comment:
I think all the functions in this file can end up in the querier. They don't look to be used anywhere else?